build.gradle

```diff
@@ -23,6 +24,8 @@ tasks.withType(Test) {
         maxFailures = 10
         failOnPassedAfterRetry = true
     }
+    classpath = files(configurations.testRuntimeClasspath)
+    classpath.from(sourceSets.test.output)
```

Comment:
I think this needs some explanation. Can you add some comments describing why we need this?
Also, why not add the two entries the same way? Say:

```groovy
classpath = files(configurations.testRuntimeClasspath, sourceSets.test.output)
```

...or perhaps:

```groovy
classpath = files()
    .from(configurations.testRuntimeClasspath)
    .from(sourceSets.test.output)
```

Reply:
Done.